modules/parsers/parser-hdbschema/com.sap.xsk.parser.hdbschema/src/main/java/com/sap/xsk/parser/hdbschema/models/XSKHDBSCHEMADefinitionModel.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.parser.hdbschema.models;
+
+import com.sap.xsk.parser.hdbschema.exceptions.XSKHDBSchemaMissingPropertyException;
+import java.util.List;
+import java.util.Objects;
+
+public class XSKHDBSCHEMADefinitionModel {
+
+  private String schema;
+
+  public String getSchema() {
```

Comment:
Use lombok plugin for automatic  generation of the getters and setters

 <dependency>
            <groupId>org.projectlombok</groupId>
            <artifactId>lombok</artifactId>
            <version>${org.projectlombok.version}</version>
            <scope>provided</scope>
        </dependency>